e2e/cypress/integration/dashboard-tus.spec.ts

```diff
@@ -45,8 +45,9 @@ describe('Dashboard with Tus', () => {
 
   it('should upload remote image with URL plugin', () => {
     cy.get('[data-cy="Url"]').click()
-    cy.get('.uppy-Url-input').type('https://via.placeholder.com/600x400')
+    cy.get('.uppy-Url-input').type('https://raw.githubusercontent.com/transloadit/uppy/main/assets/developed-by-transloadit.png')
```

Comment:
Can we make it fetch `127.0.0.1` instead? That would avoid network flakiness, and make the test pass locally even if I don't have internet.

Reply:
I'm not sure how to make this work. When I run it locally I don't think anything refers to that IP?

Reply:
I think it can be solved by starting a simple local server to host the file, something like this:
```js
require('http').createServer((req, res) => fs.createReadStream('/path/to/file').pipe(res))
```